src/components/ClaimedBooks/ClaimedBooks.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+import BookListing from "../Booklisting/Booklisting"
+
+function ClaimedBooks() {
+    const [books, setBooks] = useState ([])
+    const [clickedBookId, setclickedBookId] = useState(null)
```

Comment:
This state isn't being used so it should be removed if it isn't needed.

Reply:
e6cf020